distribution/docs/src/main/resources/content/_integrating/_endpoints/catalog-rest-endpoint.adoc

```diff
@@ -1,133 +1,101 @@
 :title: Catalog REST Endpoint
 :type: endpoint
 :status: published
+:operations: ingest, retrieval, crud
 :link: _catalog_rest_endpoint
-:summary: Allows clients to perform CRUD operations on the Catalog using REST, a simple architectural style that performs communication using HTTP.
+:url: \${public_url}/services/catalog
+:summary: Allows clients to perform operations on the Catalog using REST, a simple architectural style that performs communication using HTTP.
 :implements: https://www.w3.org/2001/sw/wiki/REST[REST specification] {external-link}.
 
-The Catalog REST Endpoint allows clients to perform CRUD operations on the Catalog using REST, a simple architectural style that performs communication using HTTP. 
+== {title}
 
-===== Installing the Catalog REST Endpoint
+The Catalog REST Endpoint allows clients to perform operations on the Catalog using REST, a simple architectural style that performs communication using HTTP. 
 
-The Catalog REST Endpoint allows clients to perform CRUD operations on the Catalog using REST, a simple architectural style that performs communication using HTTP. 
-
-The URL exposing the REST functionality is located at \${secure_url}/services/catalog.
-
-===== Installing the Catalog REST Endpoint
-
-The Catalog REST Endpoint is installed by default with a standard installation in the ${ddf-catalog} application.
-
-===== Configuring the Catalog REST Endpoint
-
-The RESTful CRUD Endpoint has no configurable properties.
-It can only be installed or uninstalled.
+.Catalog REST Endpoint Operations
+The Catalog REST Endpoint provides the capability to query, create, update, and delete metacards and associated resource in the catalog provider as follows:
```

Comment:
✏️ With the way you put more information below this, it seems this should end with `in the catalog provider.` and there should be another sentence right before the table below that say something like `Operations on the REST endpoint can be performed as follows:`

Reply:
that makes sense. fixed.